pkg/ruler/ruler.go

```diff
@@ -28,8 +28,9 @@ import (
 	"github.com/cortexproject/cortex/pkg/ingester/client"
 	"github.com/cortexproject/cortex/pkg/ring"
 	"github.com/cortexproject/cortex/pkg/ring/kv"
-	"github.com/cortexproject/cortex/pkg/ruler/rules"
-	store "github.com/cortexproject/cortex/pkg/ruler/rules"
+	"github.com/cortexproject/cortex/pkg/ruler/rulespb"
+	store "github.com/cortexproject/cortex/pkg/ruler/rulespb"
```

Comment:
**suggestion:** we may be able to trim this down

```suggestion
	"github.com/cortexproject/cortex/pkg/ruler/rulespb"
```

Reply:
Definitely! I didn't notice it, thanks! (and done)